### DISCUSSION TITLE
E03 error

### DISCUSSION THREAD

**Discussion by Mat-Moo** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#807] [Q&A]
So weird one, everything I've found indicated an E03 error is ambient temperature issue... buts its been 12c today and the water is about 12c as well. I've pulled it all out and had it in the sun, reconnected, ran for a minute or so then showed the same error again. I've tried the thermal reset switch, and various other things - but still after running with pump and heater on, after a couple of mins, comes back up with an E03 error. Ideas welcome :(

**Comment by visualapproach** [comment]
Pump model? Did it work before? PCB? FW version?

**Reply by Mat-Moo** [reply]
Pump has been fine for a few years, had mod on for a year or so, all good - just started the other day... 

So my suspicion is the temperature sensor has gone faulty. I've tried resetting it, but it just comes back again - I don't think it's fault of this software (although I need to update my version), so was more a generic question.

**Reply by visualapproach** [reply]
Ok, just to be sure you could check if it throws the error without the device being connected

**Comment by Mat-Moo** [comment]
Replaced temperature sensor - been running about an hour so far without issue
